src/important-links.md

```diff
@@ -0,0 +1,12 @@
+# Ignore me
+
+This file is a collection of links that are not link-checked by anyone else,
+but we want them to work. For example, the stabilization guide link is often
+posted to tracking issues on GitHub, which might break later if the guide is
+changed.
+
+[Skip this file. Or read it; I'm book, not a cop.](https://imgur.com/gallery/mSHi8)
+
+[1](https://forge.rust-lang.org/stabilization-guide.html)
+[2](https://rust-lang.github.io/rustc-guide/stabilization_guide.html#documentation-prs)
+[3](https://forge.rust-lang.org/stabilization-guide.html#updating-documentation)
```

Comment:
Interesting grouping.

Also, can this really link-check github.io links to the rustc guide...?

Reply:
I updated the file to check both the forge and guide links. Unfortunately, `mdbook linkcheck` doesn't check that anchors exist, but I'm hoping that in the future such functionality might exist... At the very least, if the guide moves again or something, we will know about it.